styles.css

```diff
@@ -95,11 +108,180 @@
     width: 2.5rem;
     height: 2.5rem;
     border-radius: 2rem;
-    background-color: white;
+    background-color: #fff;
     border: 1px solid #dfe1e6;
   }
 
   #homepage button i {
     font-size: 1.1rem;
   }
+
+  #works-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    gap: 1rem;
+    margin-bottom: 3rem;
+  }
+
+  #works-section h2 {
+    color: #172b4d;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    font-weight: 700;
+  }
+
+  #works-section hr {
+    width: 3rem;
+    height: 0.25rem;
+    background-color: #36b37e;
+    border-radius: 1.5rem;
+    margin-bottom: 3rem;
+  }
+
+  .works-container {
+    display: grid;
+    grid-gap: 1.25rem;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #dfe1e6;
+    border-radius: 0.5rem;
+    width: 20.125rem;
+  }
+
+  .work .img-placeholder {
+    width: 20rem;
+    height: 15rem;
+    background-color: #ebf0ee;
+  }
+
+  .work-info {
+    display: flex;
+    flex-direction: column;
+    padding: 1rem;
+    gap: 1rem;
+  }
+
+  .work-info h4 {
+    font-size: 1.5rem;
+    font-weight: 600;
+    line-height: 2rem;
+    max-width: fit-content;
+  }
+
+  .button {
+    text-decoration: none;
+    background-color: #36b37e;
+    padding: 0.75rem;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: #fff;
+    letter-spacing: 0.03em;
+    text-align: center;
+    margin: 0.75rem auto;
+  }
```

Comment:
- [ ] Dear @NicolasAndrehh, Please make sure to follow the button styling: `Active`, `Hover `, and `Pressed` for all your buttons. I can see you have a hover effect but it's not enough and not according to the Figma design. 

![Screenshot 2023-01-13 092215](https://user-images.githubusercontent.com/69077061/212381413-1663b9b9-f82a-4ceb-aee9-7fab5703669f.png)